assets/src/Routes.tsx

```diff
@@ -27,10 +28,18 @@ const RoomPageWrapper: React.FC = () => {
   );
 };
 
+const HomePageWrapper = () => {
+  const { state } = useLocation();
+  const isLeavingRoom = !!state?.isLeavingRoom;
+  const roomId = state?.roomId;
```

Comment:
Is it possible that the roomId is not set? How would the app behave then? Shouldn't it be handled?